resolvers/donationResolver.ts

```diff
@@ -219,7 +219,8 @@ export class DonationResolver {
             toWalletAddress: toAddress.toString().toLowerCase(),
             fromWalletAddress: fromAddress.toString().toLowerCase(),
             donationValueUsd: donation.valueUsd,
-            donationValueEth: donation.valueEth
+            donationValueEth: donation.valueEth,
+            verified: project.verified != null ? project.verified : ''
```

Comment:
I think it's better to set a `boolean` value for `verified` field and you can write this line shorter like this:
`verified: project.verified` (project.verified is boolean)

Reply:
my only intention is that if we hit some weird edge cases where the project doesn't have a verified property that we can assign something to it, preventing it from breaking.

Perhaps instead of an empty string `' '` we change it fo `false`? Or if we're absolutely confident we can simply keep it as `verified: project.verified` as you suggested.

Reply:
As I know we have it always but because I'm not 100% sure, so you can set something like this (just for having simpler code)
`verified: Boolean(project.verified)`

PS: Sorry for the requested changes because your functionality was ok from the beginning, I think it is worth putting some time to write cleaner code ( but anytime you thought I'm obsessed with small things just tell me :)  )